tools/system_libs.py

```diff
@@ -1021,8 +1021,11 @@ def get_files(self):
     else:
       return [shared.path_from_root('system', 'lib', 'pthread', 'library_pthread_stub.c')]
 
-  def get_base_name_prefix(self):
-    return 'libpthreads' if self.is_mt else 'libpthreads_stub'
+  def get_base_name(self):
+    name = super(libpthreads, self).get_base_name()
+    if self.is_mt:
+      name += '_stub'
```

Comment:
Wait this doesn't make sense. `libpthreads` is already `MTLibrary`. We should just delete `get_base_name_prefix`.

Reply:
Oh I see.. libpthreads is different to other libraries that are build with threading and without.